src/packages/@ncigdc/modern_components/ClinicalAnalysis/ClinicalVariableCard/index.js

```diff
@@ -1509,11 +1509,12 @@ export default compose(
       },
     })
   ),
-  withProps(({ data, fieldName, variable }) => {
+  withProps(({ data: { explore }, fieldName, variable }) => {
```

Comment:
fix issue with `data.explore.cases` crashing the app when `explore` isn't available